cmd/commands/shared.go

```diff
@@ -27,4 +27,5 @@ const (
 or 'secretKeyRef' to select a key from a Secret. The following formats are supported:
   --env-from configMapKeyRef:{config-map-name}:{key-to-select}
   --env-from secretKeyRef:{secret-name}:{key-to-select}`
+	verboseUsage = "print details of command progress"
```

Comment:
should this '=' line up with the ones above? Seems that is the style used here and in other places. Not sure what the typical Go formatting for this is.

Reply:
@trisberg It's working as designed! There is one standard formatting rule, defined by `go fmt`, but sometimes it's a bit too subtle and looks wrong. See [this comment](https://github.com/golang/go/issues/7819#issuecomment-66093841)  about a similar issue to get a flavour...

Reply:
@trisberg It's working as designed! There is one standard formatting rule, defined by `go fmt`, but sometimes it's a bit too subtle and looks wrong. See [this comment](https://github.com/golang/go/issues/7819#issuecomment-66093841)  about a similar issue to get a flavour...

Reply:
go figure :) thanks for the link